deploy-source/apps/expense-service/src/main/docker/Dockerfile.jvm

```diff
@@ -1,54 +1,92 @@
 ####
 # This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
 #
-# Before building the docker image run:
+# Before building the container image run:
 #
-# mvn package
+# ./mvnw package
 #
 # Then, build the image with:
 #
-# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/expense-restful-service-jvm .
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/expense-client-jvm .
```

Comment:
These are just comments but you might want to keep the `expense-service-jvm` name (since it's the `expense-service` app)

```suggestion
# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/expense-service-jvm .
```

Reply:
(probably better to do `sed` than me finding all tokens of those)